index.js

```diff
@@ -131,6 +131,41 @@ Limiter.prototype.get = function (fn) {
 };
 
 /**
+ * reset rate limit
+ *
+ * @api public
+ */
+Limiter.prototype.reset = function(fn){
+  
+  var count = this.prefix + 'count';
+  var limit = this.prefix + 'limit';
+  var reset = this.prefix + 'reset';
+  var duration = this.duration;
+  var max = this.max;
+  var db = this.db;
+  
+  function recreate() {
```

Comment:
please tell me what would you expect from a reset function. Why creating a new `Limiter` is not enough? 

ultimately I'd like reuse of code, if there is no other option we'll go with your solution. What do you think?